atomica/results.py

```diff
@@ -54,6 +54,47 @@ def t_observed_data(self):
     def pop_labels(self):
         return [x.label for x in self.model.pops]
 
+    def get_alloc(self):
+        # Return a dict with time-varying funding allocation at every time point in the simulation
+        if self.model.progset is None:
+            return None
+        else:
+            return self.model.progset.get_alloc(self.t, self.model.program_instructions)
+
+    def get_coverage(self,quantity='fraction'):
```

Comment:
this should eventually have a `sample` argument also

Reply:
hmm this is in `results.py` so it's returning whatever was used in a completed model run, whereas the sampling needs to take place during `model.py` so I think the `sample` argument only needs to be present in the functions in `programs.py`. Actually something we might need to do is to store the coverage values rather than having `result.get_coverage()` recompute them (and just take the storage hit) if we want to record exactly what sampled values were actually used to generate a simulation...I'll make a trello card so we remember that